### ISSUE TITLE
[Feature Request] Pipe trick for suffix without pipe character

### ISSUE THREAD

**Issue by marcjulianschwarz** [marcusolsson/obsidian-pipe-tricks#6]
Hi @marcusolsson,
Thank you for the great plug-in, I really enjoy using it.

Thats why I have a feature request. As you stated in the README it should be possible to write `[[pipe]]s` and have it convert to `[[pipe | pipes]]`. However right now, you have to use the pipe symbol like this:  `[[pipe|]]s` to get this behavior. 

Would it be possible to implement it like this `[[pipe]]s -> [[pipe | pipes]]`?
I think this would make it even easier to create good links.

Thank you.
Kind regards,
Marc Julian

**Comment by marcusolsson** [comment]
Yes. I'd even be inclined to call this a bug, as it's very much an intended functionality. Thanks for letting me know!

**Comment by etiennepellegrini** [comment]
In the same vein, do you think it'd be possible to enhance the suffix trick to work with apostrophes?
A common use case for the suffix is with possessives: I'd love to be able to have `[[pipe]]'s -> [[pipe | pipe's]]`

Thanks!

**Comment by marcjulianschwarz** [comment]
Has this been solved/fixed yet?
With the newest version installed it still doesn't seem to work for me.
